modules/oci-image/01_mod.mk

```diff
@@ -93,11 +94,11 @@ $(oci_build_targets): oci-build-%: | $(NEEDS_KO) $(NEEDS_GO) $(NEEDS_YQ) $(bin_d
 	GOEXPERIMENT=$(GOEXPERIMENT) \
 	$(KO) build $(go_$*_source_path) \
 		--platform=$(oci_platforms) \
-		--oci-layout-path=$(CURDIR)/$(oci_layout_path) \
+		--oci-layout-path=$(oci_layout_path) \
 		--sbom-dir=$(CURDIR)/$(oci_layout_path).sbom \
 		--sbom=spdx \
 		--push=false \
-		--base-import-paths
+		--bare
```

Comment:
What difference does this flag make? I was not able to find a difference between `--base-import-paths` and `--bare`.

Reply:
I think the image name is not actually used anyway, I use the digest and publish to the configured image path that way.